library/Zend/InputFilter/InputFilterPluginManager.php

```diff
@@ -46,6 +46,7 @@ public function __construct(ConfigInterface $configuration = null)
         parent::__construct($configuration);
 
         $this->addInitializer(array($this, 'populateFactory'));
+        $this->addAbstractFactory('Zend\InputFilter\InputFilterAbstractServiceFactory');
```

Comment:
Remove this; we'll document how to enable it, but not all users of the `InputFilterPluginManager` will necessarily want to use this abstract factory.